### PR TITLE
Add title to `YoutubeAtom` button label and iframe

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?

This PR addresses issues raised in the DAC report by adding an `aria-label` to the `YoutubeAtom` overlay `play` button, and a `title` to the iframe.

DAC Issues:
[Ensure all <frame> and <iframe> elements have valid title attribute values.
](https://drive.google.com/file/d/1ZXsQqmImEIx0cirPPRsaX1LKRCeoPZp5/view)

[For each instance of this ‘Play Video’ button, the video that it is in relation should be included within
the button text.](https://drive.google.com/file/d/1hBH7i4qJVeh6Wu_laQ5FLzmL_IZbAjIe/view)

### CODE REVIEW

**Review by abeddow91 — APPROVED** [review]
Nice works

File: src/YoutubeAtom.tsx [hunk 1]
```diff
@@ -160,6 +160,7 @@ export const YoutubeAtom = ({
                         role={role}
                         duration={duration}
                         pillar={pillar}
+                        title={title}
```
Comment (arelra) [inline src/YoutubeAtom.tsx:163]: Do we also need to update `YoutubeAtom.test.tsx`?

Comment (joecowton1) [reply src/YoutubeAtom.tsx:161]: Added tests for `aria-label` and `title`

**Review by arelra — APPROVED** [review]
👍

**Pull request merged**